modular_skyrat/master_files/code/_globalvars/maint_loot_oddity.dm

```diff
@@ -65,5 +65,10 @@ GLOBAL_LIST_INIT(oddity_loot, list(//oddity: strange or crazy items
 	/obj/item/storage/toolbox/syndicate = 100,
 	/obj/item/storage/toolbox/infiltrator = 100,
 	/obj/item/toy/sprayoncan = 200,
-	/obj/item/warpwhistle = 5
+	/obj/item/warpwhistle = 5,
+	/obj/item/spear/grey_tide = 1,
```

Comment:
No, this has absolutely no business being obtainable in round, grey_tide is one of the strongest items ingame.

Reply:
this is fine if you guys didn't make maintenance loot way more available via trash piles